tests/test_statistics.py

```diff
@@ -8,7 +8,7 @@
 
 import dpnp
 
-from .helper import get_all_dtypes
+from .helper import assert_dtype_allclose, get_all_dtypes
```

Comment:
Could you please add `test_statistics.py` and `test_meanvar.py` to `TEST_SCOPE` of `.github/workflows/conda-package.yml`?